plugins/optimization-detective/class-od-html-tag-processor.php

```diff
@@ -179,13 +179,14 @@ final class OD_HTML_Tag_Processor extends WP_HTML_Tag_Processor {
 	private $buffered_text_replacements = array();
 
 	/**
-	 * Count for the number of times next_token() was called
+	 * Count for the number of times that the cursor was successfully moved.
```

Comment:
Actually, it may not be "successful" as it could just be an attempt to move the cursor.

```suggestion
	 * Count for the number of times that the cursor was moved.
```